content/guidance/index.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: Guidance
+weight: 800
+dateAdded: 2021-01-22
+dateUpdated: 2021-02-01
```

Comment:
are these needed? I'd argue probably not

Reply:
Agree, from a content perspective they're not really needed, and we'd have a paper trail of dates from commits, right?

Reply:
makes sense, have removed these